controllers/controllers.js

```diff
@@ -0,0 +1,12 @@
+const { getCategoriesM } = require("../models/models");
+
+exports.getCategoriesC = (req, res, next) => {
+  getCategoriesM()
+    .then((categories) => {
+      console.log(categories);
```

Comment:
avoid leaving console logs in your code, we dont want any of them to be left in production and its might be a bit messy to remove them all later :)